src/test/scala/org/bblf/client/BblfshClientSupportedLanguagesTest.scala

```diff
@@ -0,0 +1,18 @@
+package org.bblf.client
+
+import gopkg.in.bblfsh.sdk.v1.protocol.generated.SupportedLanguagesResponse
+import org.bblfsh.client.BblfshClient
+import org.scalatest.{BeforeAndAfter, FunSuite}
+
+class BblfshClientSupportedLanguagesTest extends FunSuite with BeforeAndAfter {
+  val client = BblfshClient("0.0.0.0", 9432)
+  var resp: SupportedLanguagesResponse = _
+
+  before {
+    resp = client.supportedLanguages()
+  }
+
+  test("Check languages") {
+    assert(!resp.languages.isEmpty)
```

Comment:
This could fail if the bblfshd didn't have any drivers installed while not really being a failure in the client. Could you add an exec() to run the driver install command so we are sure there is at least a driver installed and then check that resp.languages contain that specific language?